src/components/dashboard/containers/Tables/toolbars.js

```diff
@@ -1,17 +1,48 @@
 import React, { Fragment } from 'react'
-import IconButton from '@material-ui/core/IconButton'
-import Tooltip from '@material-ui/core/Tooltip'
-import RefreshIcon from '@material-ui/icons/Refresh'
+import { IconButton, Tooltip, Box, Button } from '@material-ui/core'
```

Comment:
This kind of import should not be used without [`babel-plugin-import](https://material-ui.com/guides/minimizing-bundle-size/#option-2). Configure it or not use it.